packages/serverless-contracts/package.json

```diff
@@ -46,6 +46,7 @@
     "@types/aws-lambda": "^8.10.109",
     "@types/http-errors": "^2.0.1",
     "ajv": "^8.11.2",
+    "aws-cdk-lib": "^2.64.0",
```

Comment:
Having the dependency here will make everyone download the lib, even if they do not use it.

It will also mean that consumers of the package, which can be frontend applications, will install this dependency.

We might need to rethink how we structure our packages, because making the producers and the consumers install dependencies that they will not use is really problematic.

In the meantime, we maybe can add it as an [`optionaDependencies`](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#optionaldependencies)?

Reply:
That is correct, and since it is only for one type, I'll simply copy it in our repo